README.md

```diff
@@ -132,6 +132,9 @@ API documentation is available <a  href="http://cxdoc.unbxd.io/search-JS-library
 | fireAnalyticsEvents | Boolean | you wish to have unbid analytics along with sdk. |
 | hashMode | Boolean | you wish the url changes on hash change |
 | updateUrls | Boolean | if you dont need url tracking make it false, by default it is true |
+| actionBtn  | String  | css class, if you wish to trigger click on your custom elements, add this class |
+| actionChange  | String  | css class, if you wish to trigger change on your custom input elements, add this class |
```

Comment:
if these are classnames, please change the config names to reflect that like `actionBtnClass`, `actionChangeClass`.